addons/binding/org.openhab.binding.facerecognition/lib/README.md

```diff
@@ -0,0 +1,4 @@
+Download required libraries
```

Comment:
When you update to the new buildsystem, please remove the "lib" directory. You can finally declare dependencies in your pom.xml file. Have a look at jcenter or maven central and find the required dependencies.